msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationKotlinTest.kt

```diff
@@ -2291,5 +2394,33 @@ class NativeAuthPublicClientApplicationKotlinTest : PublicClientApplicationAbstr
         val result = application.signUp(invalidUsername)
         assertTrue(result is SignUpError)
         assertTrue((result as SignUpError).isInvalidUsername())
+
+        checkSafeLogging()
+    }
+
+    private fun checkSafeLogging() {
+        val piiTrueToCheck = listOf(
+            "password:","password=",
+            "code:","code=",
+            "continuationToken:","continuationToken="
+        )
+        val piiFalseToCheck = listOf(
+            "username:", "username="
+        )
+
+        val elementsToCheck = piiTrueToCheck.toMutableList()
+
+        if (!allowPII) {
+            elementsToCheck.addAll(piiFalseToCheck)
+        }
+
+        elementsToCheck.forEach { element ->
+            verifyLogDoesNotContain("Command", element)
```

Comment:
Why do we only care about TAG=Command? I think we should make the tag matcher `any()`, as we don't care what the tag value is in this context.

Reply:
Narrowing down the tag contributes to easier regex definition. Modify from "Command|Indicator" to any()